Source/Extensions/Blazorise.SignaturePad/SignaturePad.razor.cs

```diff
@@ -275,7 +275,7 @@ public async Task NotifyBeginStroke()
     [Parameter] public bool IncludeImageBackgroundColor { get; set; }
 
     /// <summary>
-    /// If true, prevents the user interactions and make it appear normal.
+    /// If true, prevents the user interactions.
```

Comment:
Actually might be "better english" like this?
`"If true, prevents the user interaction."`

or 

`"If true, prevents user interaction."`

But at least **interaction** instead of **interactions**? or do you think the latter is more appropriate?